driver/terraform.go

```diff
@@ -119,7 +119,12 @@ func NewTerraform(config *TerraformConfig) (*Terraform, error) {
 		for k, v := range taskEnv {
 			env[k] = v
 		}
-		tfClient.SetEnv(env)
+		err = tfClient.SetEnv(env)
+		if err != nil {
+			logger.Error("error setting the environment for the client",
+				"client_type", config.ClientType, "error", err)
+			return nil, err
```

Comment:
Since we weren't checking this error before... maybe I should just log a warning and then do nothing? (not return nil,err). I think it would still be good to check for the error returned here

Reply:
I'm thinking it's fine to error out now because if setting the environment variables fails, then the terraform-exec won't behave as expected.